### ISSUE TITLE
Feature Proposal: Add sub-dashboards per spoken language

### ISSUE THREAD

**Issue by domcorvasce** [thepracticaldev/dev.to#247]
# _Feature_
### Request or User Story

Well, there are a lot of users on dev.to which speak different languages and come from different countries.
What about adding some sort of sub-dashboards grouped by spoken language where users can interact with each other using their native language?

### Definition of Done

I should be able to choose where post my article:

  - on the English dashboard (the main one)
  - on the X dashboard (like dev.to/it or dev.to/fr)

### Notes

Although I think this feature will allow users to meet other users in the same city or nation encouraging meetups and so on — I'm worried users could choose to post articles only on their sub-dashboard and not on the main one.

**Comment by pkfrank** [comment]
Hi @domcorvasce, thanks for the issue.

We're definitely interested in improving our multi-language features.  We introduced some features to support multiple languages [last November](https://dev.to/ben/changelog-language-detection-and-segmentation-266), but we plan to give this entire direction a much closer look soon.

Your suggestion about having different "dashboards" could be the right approach.  Some users have simply used the existing "tag" systems (IE for [Spanish](https://dev.to/t/spanish), [Italian](https://dev.to/t/italian), and [Japanese](https://dev.to/t/japanese)), but we'd like to support this much more natively, and without requiring explicit use of a dedicated tag.

We might also look to leverage [DEV Connect](https://dev.to/ben/changelog-announcing-dev-connect-beta-1i5f) to support this direction in some form.

Basically, we just need a little bit more time to consider our options and try to improve this set of features for multiple languages.

Thanks for offering this feedback!
